wp-content/themes/inf_theme/README.md

```diff
@@ -122,7 +122,7 @@ Once the theme is activated, make sure to run `npm start` in your `public_html`
 1. **Linting assets** - JS and SASS using Webpack:
 
     ```bash
-    npm run precommit
+    npm run lint
```

Comment:
We will change the readme before we merge to master and tag because we have husky, so manual linting is kinda not necessary, right?